### PR TITLE
tencent-pcp-623

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.1 Feature**

TKE new public security group and subnet input changes
pcp-623

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1049#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:50]: Should there be a space after colon?

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:246]:
```suggestion
    | **SSH Key Pair Name**| Choose the desired SSH keypair. You must preconfigure SSH key pairs <br /> on TKS for the<br /> desired regions. The selected key is inserted into the provisioned VMs.
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:290]:
```suggestion
In Tenant Admin and Project Admin scope, Palette allows you to force the deletion of a cluster that's been stuck in **Deletion** state for a minimum of **15 minutes**.
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:305]:
```suggestion
If any resources remain in the cloud, you should clean them up before initiating a forced delete. 
```

**Review by ritawatson — APPROVED** [review]
LGTM with comments: 
- I noticed an InfoBox with this statement that's unclear, but not sure what it's trying to say: "Palette encourages its uses to go with the Public Cluster endpoint access as of now. Other options will be supported in the near future."

- In the sentence below, what is meant by "clean them up"? If it means to delete those resources, we should say delete. But then it doesn't make sense to delete something before a force delete? 

"If any resources remain in the cloud, you should **clean them up** before initiating a forced delete."

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1049#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1049#pullrequestreview-1221390987" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:292]:
```suggestion
```
We don't need this heading anymore.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:47]: @resmaa these prerequisites should not be in number order but instead as an ordered list.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:50]:
```suggestion
- Inbound traffic rule that allows the TCP/HTTPS protocol for port 443 from all IPv6 and IPv4 sources.
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:47]: This is a very vague statement, and it might be best to remove it if we don't have specific guidance to offer. Otherwise it can lead to confusion.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:157]:
```suggestion
1. Log in to the Palette and from the **Tenant Admin Settings**, select the **Cloud Accounts** tab.
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:160]:
```suggestion
2. Click **+ Tencent Account** to open the cloud account creation wizard and fill in the following details:
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:176]:
```suggestion
**Note**: The cloud account can be created during the first step of cluster creation when you fill in the basic information by clicking the **+** next to **Cloud Account**. 
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:183]:
```suggestion
   * **Name**, **Description**, and **Tags**. Tags on a cluster are propagated to the VMs deployed on the cloud or data center environments. 
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:184]:
```suggestion
   * Select the desired [Tencent cloud account](/clusters/public-cloud/tke#createatencentcloudaccount). 
```
The later can be removed because it's a prerequisite stated earlier.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:186]: This can be removed. We already stated it.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:246]:
```suggestion
     | **SSH Key Pair Name**| Choose the desired SSH keypair. You must preconfigure SSH key pairs on TKS for the desired regions. The selected key is inserted into the provisioned VMs.
```
I'm not sure I understand why we have HTML breakpoints here.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:247]:
```suggestion
    | **VPCID**|The ID of the Virtual Private Cloud (VPC) that the stack is to be launched into. The VPC must be in the specified region. All cluster instances will be launched into this VPC. |
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and changes are below. This is another page that in the future we need to breakout as there is way too much happening on a single page.

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:184]:
```suggestion
   * Select the desired [Tencent cloud account](/clusters/public-cloud/tke#createatencentcloudaccount). The Tencent credentials must be pre-configured in the Project/Tenant Admin settings.
```

File: content/docs/04-clusters/01-public-cloud/06-tke.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/06-tke.md:272]: @resmaa take a peek at how I created Delete a Cluster for the AWS and Azure section [[Preview](https://deploy-preview-1058--docs-spectrocloud.netlify.app/clusters/public-cloud/aws/create-cluster#deletinganawsiaascluster)] [[PR](https://github.com/spectrocloud/librarium/pull/1058)]

Comment (resmaa) [reply content/docs/04-clusters/01-public-cloud/06-tke.md:272]: nice

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1049#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1049#pullrequestreview-1221390987" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | f49935126f0003671c7c1a9a393c98fe2aa2e563 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1049/librarium_build/1606166994736910336) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1049#pullrequestreview-1229166573" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1049#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1049#pullrequestreview-1221390987" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
